src/compiler/scala/tools/nsc/settings/ScalaSettings.scala

```diff
@@ -264,6 +264,7 @@ trait ScalaSettings extends StandardScalaSettings with Warnings { _: MutableSett
 
   val Youtline        = BooleanSetting    ("-Youtline", "Don't compile method bodies. Use together with `-Ystop-after:pickler` to generate the pickled signatures for all source files.").internalOnly()
 
+  val unsafe = MultiStringSetting("-Yunsafe", "packages", "Expose platform packages under --release")
```

Comment:
Maybe the name is a bit too generic? I see the relation to `misc.unsafe`, but still.

Reply:
`-Ymisc-unsafe`? `-Yunsafe-misc`? Don't make me break open `-Zmisc.unsafe`. Maybe Seth will have an idea.

Reply:
`-Yopen-packages`